djimporter/fields.py

```diff
@@ -254,7 +254,12 @@ def to_python(self, value):
         except ObjectDoesNotExist:
             msg = "No match found for %(model)s with value %(value)s"
             params = {'model': self.model.__name__, 'value': value}
-            raise ValidationError(msg, params=params)
+
```

Comment:
Maybe you can do it before the try.  Not need to try to find the object value if value ==''. It can raise  if value not found but not ''